docs/sphinx/developers/examples/bftest.m

```diff
@@ -0,0 +1,169 @@
+bfCheckJavaPath();
+
+% Create local file for testing
+java_tmpdir = char(java.lang.System.getProperty('java.io.tmpdir'));
+uuid = char(java.util.UUID.randomUUID());
+tmpdir = fullfile(java_tmpdir, uuid);
+
```

Comment:
Absolutely, this is likely the same issue as the one reported in https://ci.openmicroscopy.org/view/DEV/job/BIOFORMATS-DEV-merge-matlab/11/console.